homework/src/index.js

```diff
@@ -33,14 +33,123 @@
   function main(url) {
     fetchJSON(url, (err, data) => {
       const root = document.getElementById('root');
+      const header = createAndAppend('header', root);
+      createAndAppend('p', header, {html: 'HYF Repositories', class: 'header-lable'});
+      const select = createAndAppend('select', header, { id: 'selectMenu' });
       if (err) {
         createAndAppend('div', root, { html: err.message, class: 'alert-error' });
       } else {
-        createAndAppend('pre', root, { html: JSON.stringify(data, null, 2) });
+        //createAndAppend('pre', root, { html: JSON.stringify(data, null, 2) });
+        /* &&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&& */
+        data.sort((a, b) => a.name.localeCompare(b.name, 'fr', {ignorePunctuation: true}));
+        data.forEach((reopsitory, index) => {
+          createAndAppend('option', select, {html: reopsitory.name, value: index});
+        });
+        /* &&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&& */
+        const container = createAndAppend('div', root, { id: 'container'});
+        contentLeft(container, data, select);
+        contentRight(container, data, select);
       }
     });
   }
 
+
+
+
+
+  // function who creates a left content section
+  function contentLeft(container, data, select) {
+    const leftSection = createAndAppend('div', container, { id: 'leftSection', class: 'whiteframe' });
+    showTableRepo(leftSection, data, 0);
+
+    select.addEventListener('change', (event) => {
+      document.getElementById('leftSection').innerHTML = "";
```

Comment:
There is no need to use `getElementById` to get access to the `leftSection` element. It is still available to you from the variable in line 62. Review the concept of JavaScript closures again.

Reply:
Oh oké i know it but i did that first just to check that contentLeft function works good